packaging/CMakeLists.txt

```diff
@@ -2,19 +2,21 @@
 # OS-independent CPack configuration.
 set(CPACK_COMPONENTS_ALL "")
 set(CPACK_OUTPUT_FILE_PREFIX "dist")
+set(CPACK_PACKAGE_NAME "ZeekAgent")
 set(CPACK_PACKAGE_CONTACT "info@zeek.org")
 set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Zeek Agent is an endpoint monitoring tool reporting host information to Zeek.")
 set(CPACK_PACKAGE_HOMEPAGE_URL "https://zeek.org")
 set(CPACK_PACKAGE_RELOCATABLE yes)
 set(CPACK_PACKAGE_VENDOR "Zeek Project")
-set(CPACK_PACKAGE_VERSION "${ZEEK_AGENT_VERSION}")
```

Comment:
I don't know/remember if this is used anywhere by CPack. Are you positive it isn't?

Reply:
Ah, didn't mean to remove that. It has to be set to something different for the Windows installer, but that line there should remain.